kfdefs/overlays/moc/smaug/seldon/seldon-podmonitor.yaml

```diff
@@ -3,9 +3,7 @@ kind: PodMonitor
 metadata:
   name: seldon-podmonitor
 spec:
-  selector:
-    matchLabels:
-      app.kubernetes.io/managed-by: seldon-core
+  selector: {}
   podMetricsEndpoints:
     - port: metrics
       path: /prometheus
```

Comment:
Why is the namespace selector below pointing to any ?

Reply:
I think it will match entire namespace by default, there can be other services with different label where we want to get metrics in the same namespace, leaving empty will help to discover metrics in entire namespace otherwise we get restricted to specific service.

Reply:
If the namespace matches are not correct, targets won't show up. And it is also consistent with the namespace selectors defined for the servicemonitors for other services (ex: jupyterhub[https://github.com/operate-first/apps/blob/master/kfdefs/overlays/moc/smaug/jupyterhub/servicemonitor.yaml](url))

Reply:
my concern is that the namespaceselector is set to any , so it loooking at whole cluster
if we remove the selector per label, then it would pick up all the metrics in the whole namespace.
which could causes if in future, while debugging metrics. 

i would suggest: either restrict namespace to seldon or selector based on seldon label

Reply:
Leaving namespace selector to empty will help to get all metrics in the same namespace, in this way we can monitor entire namespace